### PR TITLE
Create Morse Decoding App - Decode Morse Project

### PR DESCRIPTION
**Pull request by ouasamine**
## Overview
In this first part, we set up the project base files and created 3 methods that take Morse codes as parameters and convert it to understandable speech.

## Workflow
- Set up files (ruby file, README, linters)
- Create the methods

### CODE REVIEW

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hello  @ouasamine and @arabiu033                           :wave:   
 Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job with your methods, they are almost doing their job:+1: 

 ## Required Changes 
- [ ] Check the comment below :arrow_double_down: 




 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: morse.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  print(' ')
+end
+
+def decode(string)
+  ar = string.split(/   /)
+  ar.each { |value| decode_word(value) }
+end
+
+decode('-- -.--   -. .- -- .')
+decode('.-   -... --- -..-   ..-. ..- .-.. .-..   --- ..-.   .-. ..- -... .. . ...')
```
[the remark this answers is not in the document]
Comment (arabiu033) [reply morse.rb]: Ok. Let's fix that. Thank you for the review :+1:

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED 🍾✔🎉**
Hello @ouasamine and @arabiu033 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- You documented your work professionally.✔
- Your linters are passing all the checks
- You asked for the changes in a polite and friendly way.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by arabiu033** [comment]
Hi @emmanuellekamwa,

Thank you for the review :smiley:

**Pull request merged**